packaging/rpm/alvr.spec

```diff
@@ -1,35 +1,52 @@
 Name: alvr
-Version: 16.0.0-rc1
-Release: 1.0.0
+Version: 16.0.0
+Release: 0.0.1rc1
 Summary: Stream VR games from your PC to your headset via Wi-Fi
 License: MIT
-Source: https://github.com/alvr-org/ALVR/archive/refs/tags/v%{version}.tar.gz
+Source: https://github.com/alvr-org/ALVR/archive/refs/tags/v16.0.0-rc1.tar.gz
+URL: https://github.com/alvr-org/ALVR/
 ExclusiveArch: x86_64
-BuildRequires: alsa-lib-devel cairo-gobject-devel cargo clang-devel ffmpeg-devel gcc gcc-c++ ImageMagick libunwind-devel rust rust-atk-sys-devel rust-cairo-sys-rs-devel rust-gdk-sys-devel rust-glib-sys-devel rust-pango-sys-devel vulkan-headers vulkan-loader-devel
+BuildRequires: alsa-lib-devel cairo-gobject-devel cargo clang-devel ffmpeg-devel gcc gcc-c++ ImageMagick libunwind-devel openssl-devel rpmdevtools rust rust-atk-sys-devel rust-cairo-sys-rs-devel rust-gdk-sys-devel rust-glib-sys-devel rust-pango-sys-devel selinux-policy-devel vulkan-headers vulkan-loader-devel
```

Comment:
Is there no build-time/run-time dependency distinction in RPM?

Reply:
There is, under the `Required.*` section:
https://github.com/alvr-org/ALVR/pull/753/files/8692ecf18216c4b08f01d6b3f2bf92ddebaff69f..9ed27cb36968128540d1cf540ec356b695a5c4c5#diff-8d931fdd4015e5915d60941818500aafb7d96389ada43f510502560a86918fd3L10-L12